### ISSUE TITLE
forseti server v2.8.0 leaks tmp files, eventually fills disk

### ISSUE THREAD

**Issue by rvandegrift** [GoogleCloudPlatform/forseti-security#2411]
Something in forseti server creates & deletes temp files without closing them:
```sh
$ sudo lsof -n | grep '^forseti.*deleted'
forseti_s  2967  6529          ubuntu  120u      REG                8,1 213838630       5059 /tmp/tmpiHs9vc (deleted)
forseti_s  2967  6529          ubuntu  121u      REG                8,1     98284       5062 /tmp/tmp_MpkXs (deleted)
forseti_s  2967  6529          ubuntu  122u      REG                8,1 213838630       5065 /tmp/tmp7HhFfK (deleted)
forseti_s  2967  6529          ubuntu  123u      REG                8,1     98284       4461 /tmp/tmp6t3NIP (deleted)
...
```
Since some of the files are ~200MiB, this fills the root filesystem in a few days.  Immediate fix is to restart forseti-server.

**Comment by ahoying** [comment]
This fix will be back ported to 2.8 and 2.9 and once that is done, a reboot of effected systems will pick up the patch and no longer have this issue. Thank you @rvandegrift for tracking down the issue and including the output from lsof!

**Comment by ahoying** [comment]
Fixed and backported.
